backend/python/app/__init__.py

```diff
@@ -50,7 +51,10 @@ def create_app(config_name):
     app.config["MONGODB_URL"] = os.getenv("MG_DATABASE_URL")
 
     # required for auth
-    firebase_admin.initialize_app()
+    cred = credentials.Certificate("./firebaseServiceAccount.json")
+    firebase_admin.initialize_app(
+        cred, {"storageBucket": os.getenv("DEFAULT_BUCKET")}
```

Comment:
Minor thing, but is it possible to initialize this without `cred`? The path to the private key (./firebaseServiceAccount.json) is already set as an env variable so it should automatically load the credentials